recipe/conda_build_config.yaml

```
@@ -23,7 +23,7 @@ aws_sdk_cpp:
   - "1.10.57"
   - "*"
 azure_core_cpp:
-  - "1.10.2"
+  - "1.11.0"
   - "*"
```

My previous remark was not clear enough.
```suggestion
  - "1.10.2"
  - ">=1.11.0""
```

---

It is still installing the older version [here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=961430&view=logs&j=9c5ef928-2cd6-52e5-dbe6-9d173a7d951b&t=20c71c51-4b27-578b-485d-06ade2de1d00&l=1032)

---

Yes, this is normal and expected for the `libevent-2.1.0` specification.

---

I think we have two options: (temporary) skip those builds, leaving them untouched, or overwriting these specifications.

I would go for the first one.

What do you think?

---

How will the package be published if we don't run the builds?

---

I am fine with it if the RC can be published.

---

4.5.0rc0 can be released but we can skip some the problematic ones.

---

OK. Then. Let's do that.